coffee-warehouse/src/components/Dashboard.vue

```diff
@@ -0,0 +1,313 @@
+<template>
+  <div>
+    <localization :language="localizationLanguage">

```

Comment:
not sure if we need this - we usually need just one localization provider in a project that is added high in the components' hierarchy

Reply:
Fixed with commit: https://github.com/telerik/kendo-vue/pull/164/commits/01a1e75eec6090d1ec99741070966303288ba683